DuckDuckGo/Tab/Model/Tab.swift

```diff
@@ -304,6 +303,10 @@ protocol NewWindowPolicyDecisionMaker {
             }
 
         addDeallocationChecks(for: webView)
+
+        if #available(macOS 14.4, *) {
+            WebExtensionManager.shared.eventsListener.didOpenTab(self)
```

Comment:
just asking to make sure, when testing an extension in other browsers, is this the point when the notification is being sent, i.e. on a tab initialization (or tabs restoration)?

Reply:
I'm not sure what's the point where Safari send it, since we don't have the source code. Is there any specific reason to not call it here?

Reply:
I don‘t know, just to think about… Maybe some UI related stuff that may try modifying something while the Tab isn‘t in the hierarchy yet.. 
Or the Tab may be initialized and never actually displayed (I know it shouldn‘t but technically it can)

Reply:
Yeah I think your reasoning makes sense. I moved this call to more appropriate place in `TabCollection`